src/memray/reporters/assets/common.js

```diff
@@ -192,6 +192,32 @@ function filterFramesByFunc(root, func) {
   return _.defaults({ children: children }, root);
 }
 
+function mergeNodes(root) {
```

Comment:
This function will likely need some tests in `src/memray/reporters/assets/common.test.js`